tests/templatetags/test_tags_invalid.py

```diff
@@ -0,0 +1,15 @@
+from django import template
+
+from pattern_library.monkey_utils import override_tag
+
+register = template.Library()
+
+
+@register.simple_tag()
+def default_html_tag_invalid(arg=None):
+    "Just pass, never do anything"
+    pass
```

Comment:
Wouldn't a `return None` be more explicit than a `pass`, since that returned value seems  to be the one we're actually testing there - as a representative of the non-string objects? :slightly_smiling_face: